srcpkgs/sublime-text3/template

```diff
@@ -16,12 +23,23 @@ else
 	checksum=480609962bbbd12128b5332c7831372b8851c13e160512730d1b0a6a835a3071
 fi
 
-repository="nonfree"
-archs="i686 x86_64"
-wrksrc="sublime_text_3"
-nopie=yes
+post_extract() {
+	$XBPS_FETCH_CMD https://www.sublimehq.com/eula
+	sed -n '
+		/is licensed.*not sold/p
+		/<ol>/,${ /<\/ol>/{x;p;d}; H }
+		$a </ol>
+	' <eula | w3m -dump -I utf-8 -T text/html >EULA
+
+	filesum="$(xbps-digest EULA)"
+	if [ "$filesum" != "$_license_checksum" ]; then
+		msg_error "SHA256 mismatch for EULA:\n$filesum\n"
+	fi
+}
 
 do_install() {
+	vlicense EULA
+	rm eula EULA
```

Comment:
Why is this necessary?

Reply:
a couple lines down it runs `cp --preserve=mode -r ./* "${DESTDIR}/usr/lib/sublime_text", which catches `eula` (the unprocessed HTML) and `EULA` (the text-only EULA) and puts them in the wrong place